src/ui/help.rs

```diff
@@ -1,111 +1,261 @@
-pub fn get_help_docs() -> Vec<Vec<&'static str>> {
+use crate::user_config::KeyBindings;
+
+pub fn get_help_docs(key_bindings: &KeyBindings) -> Vec<Vec<String>> {
   vec![
-    vec!["Scroll down to next result page", "<Ctrl+d>", "Pagination"],
     vec![
-      "Scroll up to previous result page",
-      "<Ctrl+u>",
-      "Pagination",
+      String::from("Scroll down to next result page"),
+      String::from("<Ctrl+d>"),
+      String::from("Pagination"),
     ],
-    vec!["Jump to start of playlist", "<Ctrl+a>", "Pagination"],
-    vec!["Jump to end of playlist", "<Ctrl+e>", "Pagination"],
-    vec!["Jump to currently playing album", "a", "General"],
     vec![
-      "Jump to currently playing artist's album list",
-      "A",
-      "General",
+      String::from("Scroll up to previous result page"),
+      String::from("<Ctrl+u>"),
```

Comment:
Scrolling up/down and jumping start/end are now added to the config - could display those values too?

This PR just got merged adding these to the config https://github.com/Rigellute/spotify-tui/pull/566